locales/en-US/translation.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I guess this file is required by i18next. Would it be possible to use a `.js` file which uses `module.export`? Then one can add comments.